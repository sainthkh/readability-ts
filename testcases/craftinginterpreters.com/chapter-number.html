<html>

<head>

</head>

<body>
    <article>
<h2><a href="#bytecode" id="bytecode"><small>14&#8202;.&#8202;1</small>Bytecode?</a></h2>
    </article>
</body>

</html>