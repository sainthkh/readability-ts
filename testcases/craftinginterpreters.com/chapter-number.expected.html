
<h2><a href="#bytecode" id="bytecode"><small>14 . 1 </small>Bytecode?</a></h2>
    